libc/src/__support/fixed_point/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ add_header_library(
     libc.include.llvm-libc-macros.stdfix_macros
     libc.src.__support.macros.attributes
     libc.src.__support.macros.optimization
+    libc.src.__support.CPP.bit
```

Comment:
You may want to update the Bazel build as well.

Reply:
Done.